lib/ex_machina.ex

```diff
@@ -96,6 +100,32 @@ defmodule ExMachina do
   end
 
   @doc """
+  Builds a factory with the passed in factory_name and returns its fields
+
```

Comment:
I think it would be good to mention that it is expecting an Ecto struct, and that it will raise an error if you try to build something other than that.